src/test/java/com/jcabi/http/wire/LastModifiedCachingWireTest.java

```diff
@@ -56,6 +59,9 @@
 
     /**
      * Test body.
+     * @todo: #120 move constants in this file to their tests because tests
+     *  must share nothing
```

Comment:
@mkordas I'm happy to raise things I notice as I understand what I'm talking about. I don't yet understand puzzle validation on Travis, so unless you give me the exact text you want to be put in this PR, I can't do this, it will just cause another day of delay for this task.